app/routes/how-can-i-help.grassroots.tsx

```diff
@@ -1,14 +1,164 @@
 import {MetaFunction} from '@remix-run/node'
+import CardSmall from '~/components/CardSmall'
+import DropDown from '~/components/DropDown'
+import {Microscope, GovermentBuilding, PuzzlePieces} from '~/components/icons-generated'
+import HelpItem from '~/components/HowCanIHelp/HelpItem'
+import CategoryCarousel from '~/components/HowCanIHelp/CatgoryCarousel'
+import Testimonial from '~/components/Testimonial'
 import Base from '~/components/HowCanIHelp/Base'
+import {useEffect} from 'react'
+import VideoImg from '~/assets/video_thumbnail.png'
+import PlayIcon from '~/components/icons-generated/Play'
 
 export const meta: MetaFunction = () => {
   return [{title: 'How Can I Help? - AISafety.info'}]
 }
 
+const TopText = () => (
+  <>
+    <div className="padding-bottom-80">
+      <div className="flexbox-alt">
+        <div className="col-6-alt">
+          <p className="default-bold padding-bottom-16">Who</p>
+          <p className="grey default">
+            For anyone—help us spread the word about this issue to friends, those in your social
+            media or professional network, or whomever.
+          </p>
+        </div>
+        <div className="col-6-alt">
+          <p className="default-bold padding-bottom-16">Why this is important</p>
+          <p className="grey default">
+            AI poses a serious risk of human extinction—potentially soon. Raising global awareness
+            is important to help us unite and take the best step forward.
+          </p>
+        </div>
+      </div>
+    </div>
+    </>
+)
+const WhatYouCanDo = () => (
+  <>
+    <h2 className="teal-500 padding-bottom-64">
+      What you can do, right now
+    </h2>
+    <div>
+      <HelpItem
+        title="Share a video or article on social media"
+        tag="2 min"
+        links={[
+          {
+            tag: 'Top recommendation',
+            title: 'A.I. - Humanity\'s Final Invention?',
+            action: 'https://youtu.be/fa8k8IQ1_X0?si=b2l0Ex6KQvt6JyQD',
+            image: (
+              <div style={{position: 'relative'}}>
```

Comment:
it might be worth making a dedicated component for this?

Reply:
Made the component but I can't get the play button to behave. That can be a CSS fix for another day